pyproject.toml

```diff
@@ -18,15 +18,17 @@ opentelemetry-instrumentation-flask = "<1.0"
 opentelemetry-instrumentation-requests = "<1.0"
 opentelemetry-exporter-otlp-proto-http = "^1.24.0"
 
-# for tests
-pytest = {version = "^8.2.0", optional = true}
-pytest-cov = {version = "^5.0.0", optional = true}
+requests = "^2.32.0"
+requests-kerberos = "^0.14.0"
 
-[tool.poetry.extras]
-test = [
-    "pytest",
-    "pytest-cov",
-]
```

Comment:
These are extra dependencies required by tests installed by `tox` (see `commands_pre` in `tox.ini`).

Reply:
okay, I thought they can be replaced by using [tool.poetry.group.dev.dependencies], I will add them back.

Reply:
Installing dev dependencies might be OK too if there are only deps for unit tests. Not sure what `--dev-dependency` parameter does, for me it is not listed with `poetry install --help`.